subprojects/gradle/performance/src/main/kotlin/com/avito/performance/PerformanceCompareAction.kt

```diff
@@ -31,13 +31,14 @@ open class PerformanceCompareAction(
             }
 
             override fun exception(msg: String, error: Throwable) {
-                logger.debug(msg, error)
+                logger.critical(msg, error)
             }
```

Comment:
@warabei14 , cc

Reply:
i'll revert this change, had no intention to change logic